app/services/ab_test_service.rb

```diff
@@ -0,0 +1,23 @@
+# This module wraps the FieldTest to provide a bit of insulation
+# between the FieldTest implementation and the application logic.
+module AbTestService
+  extend FieldTest::Helpers
+
+  # Find the appropriate feed strategy we're testing for the given user.
+  #
+  # @param user [User] find the feed strategy for the given user.
+  # @param env [Hash] container for possible ENV override of strategy.
+  #
+  # @return [ActiveSupport::StringInquirer] an inquirable string.
+  #
+  # @note You can force a named strategy by setting an ENV variable.
+  #       This forced strategy might be super useful for anyone performing
+  #       QA testing on AB Testing scenarios.
+  def self.feed_strategy_for(user:, env: ApplicationConfig, default_value: "original")
```

Comment:
Just an observation: I like the idea of injecting `env` as a dependency, but it is not a pattern we (yet) follow elsewhere.